app/livechat/client/views/app/tabbar/visitorHistory.js

```diff
@@ -67,4 +138,50 @@ Template.visitorHistory.events({
 			return instance.offset.set(instance.offset.get() + ITEMS_COUNT);
 		}
 	}, 200),
+	'keyup #searchInput': async function(event,template){ 
+		let text = event.target.value;
+		template.isChatClicked.set(false);
+		Session.set('found',false);
+	
+		if(event.target.value == ''){
+			template.isAllChat.set(true);
+		}else{
+			template.isAllChat.set(false);
+			Template.instance().searchResult = new ReactiveVar([]);
+			searchResults = [];
+
+			for(var i=0; i<allHistoryChat.length;i++){
+				if(allHistoryChat[i].msg == text){ // check search input matches with any msg
+					Session.set('found',true);
+					searchResults.push(allHistoryChat[i]); 
+					console.log(allHistoryChat[i]);
+				}
+				
+			}
+			template.searchResult.set(searchResults)
+			if(searchResults.length > 0){
+				Session.set('found',true);
+			
+			}
+		console.log(	Template.instance().searchResult.get());
+		console.log(event.target.value);	
+	}},
+	'click .list-chat': async function(event,template){
+		// console.log(TabBar.getButton('visitor-history'));
+		template.isAllChat.set(false);
+		template.isChatClicked.set(true);
+		let id = event.currentTarget.id
+		console.log(event);
+		let token = event.currentTarget.attributes.aria.value;
+		console.log('aria-lable'+token);
+				//	let historyArray = template.history.curValue;
+	//	console.log(historyArray);
+		  historyResult  =  await APIClient.v1.get(`livechat/messages.history/${ id }?token=${token}`);
+		  template.historyResu.set(historyResult.messages); 
+		// historyResult = JSON.stringify(historyResult.messages);
+		   console.log('historyResult:'+JSON.stringify(historyResult.messages));
```

Comment:
Please, remove all of this console.log code.

Reply:
Removed Console Messages